### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by cmonfortep**
Task/Issue URL: https://app.asana.com/0/0/1164365196861356/f
Tech Design URL: 
CC: 

**Description**:
Introducing Issue templates. We are creating two different templates: Bugs and feature requests. They are based on the default ones except for some minor changes to reduce their content and adapt to our platform.

These templates are intended to guide users when reporting a bug or requesting a new feature/enhancement. They will bring more consistency between all the created issues. Reduce the time labeling issues in our repo. Finally, and most important, it will try to get all the information needed to understand each request from the moment of creation.

**Steps to test this PR**:
No need to test this PR. The PR has been automatically created by Github.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Smartphone (please complete the following information):**
```
Comment (bwaresiak) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]: I'd also ask people to include information about the version of the App they are using (and possibly the steps describing how they can check that) i.e.:

 - DDG App Version: (e.g. 5.25.0)[1]
 - Device: (e.g. Pixel 2, Samsung s10)
  - OS: (e.g. Android 10)
  - If applicable, add screenshots to help explain your problem.

[1] App version can be found in...

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**How to Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
```
Comment (bwaresiak) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]: Maybe at the end add something along the lines of: 

If applicable, please include information about any settings that may be relevant to this issue (e.g. selected theme, auto clear option, etc).

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug, needs triage
+assignees: ''
+
+---
+
```
Comment (bwaresiak) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]: Shall we add a link to our contributing guidelines at the beginning?

Comment (cmonfortep) [reply .github/ISSUE_TEMPLATE/bug_report.md:9]: Not sure, I would say it's not necessary to read the contribution guidelines to file an issue. I see that more related to a real contribution.

**Comment by cmonfortep** [comment]
@bwaresiak sorry for the delay here. I added the suggestions to the template.

**Comment by CDRussell** [comment]
What's still required here to merge this?

**Comment by cmonfortep** [comment]
@CDRussell just @bwaresiak review. He will take a look today 👍

**Pull request merged**